country.rb

```diff
@@ -0,0 +1,141 @@
+module LoanEligibility
+  def loan_eligibility?
+    if @gdp<1000 && @development_status=='developing'
+      "#{@name} is eligible for IMF or World Bank loan"
+    else
+      "#{@name} is not eligible for loan"
+    end
+  end
+end
+module UnSecurityCouncil
+  def un_security_council_eligibility?
+    if @army_strength>500_000 && @gdp > 2_000_000
+      "#{@name} is eligible for UN Security Council Seat"
+    else
+      "#{@name} is not Eligible for Seat"
+    end
+  end
+end
+module WarPrediction
+  def war_outcome
+    if @army_strength >1_000_000 && @gdp >3_000_000
+      "#{@name} can likely win the war"
+      else
+        "#{@name} is likely to lose the war"
+    end
+  end
+end
+
+
+class Country
+  include LoanEligibility
+  include UnSecurityCouncil
+  include WarPrediction
+
+  attr_accessor :name,:population,:gdp,:states,:army_strength,:development_status,:region
+
+  def initialize(name,population,gdp,states,army_strength,development_status,region)
+    @name=name
+    @population=population
+    @gdp=gdp
+    @states=states
+    @army_strength=army_strength
+    @development_status=development_status
+    @region=region
+  end
+end
+$all_countries=[]
+def find_country(name)
+  $all_countries.find {|c| c.name.downcase == name.downcase}    
+end
+
```

Comment:
@PranaliM0 Instead of adding modules, classes in same file, you can also use separate files for implementation